app/team.go

```diff
@@ -1805,33 +1632,20 @@ func (a *App) RestoreTeam(teamID string) *model.AppError {
 }
 
 func (a *App) GetTeamStats(teamID string, restrictions *model.ViewUsersRestrictions) (*model.TeamStats, *model.AppError) {
-	tchan := make(chan store.StoreResult, 1)
-	go func() {
-		totalMemberCount, err := a.Srv().Store.Team().GetTotalMemberCount(teamID, restrictions)
-		tchan <- store.StoreResult{Data: totalMemberCount, NErr: err}
-		close(tchan)
-	}()
-	achan := make(chan store.StoreResult, 1)
-	go func() {
-		memberCount, err := a.Srv().Store.Team().GetActiveMemberCount(teamID, restrictions)
-		achan <- store.StoreResult{Data: memberCount, NErr: err}
-		close(achan)
-	}()
+	totalMemberCount, err := a.Srv().Store.Team().GetTotalMemberCount(teamID, restrictions)
```

Comment:
This one we should keep as-is. These queries are heavy and gives a good performance boost to keep it asynchronous.

Reply:
will see if we can optimise the underlying call on this before reverting back to the original implementation.